### PR TITLE
ADF-2944 Create a new tab in settings for AF settings

### PR DESCRIPTION
**Pull request by vprigent**

### CODE REVIEW

File: woocommerce-addressfinder.php [hunk 1]
```diff
@@ -68,13 +68,25 @@ function add_addressfinder_widget( $_checkout ) {
...
 	}
 
-	add_filter( 'woocommerce_get_settings_checkout', 'addressfinder_settings', 10, 1 );
+	// Add the tab to the tabs array
+	function filter_addressfinder_settings_tabs_array( $settings_tabs ) {
+	    $settings_tabs['addressfinder-settings'] = __( 'Addressfinder Settings', 'woocommerce' );
+
+	    return $settings_tabs;
+	}
+	add_filter( 'woocommerce_settings_tabs_array', 'filter_addressfinder_settings_tabs_array', 99 );
```
Comment (vprigent) [inline woocommerce-addressfinder.php:77]: Create a new tab and adds it to the settings tabs under woocommerce

File: woocommerce-addressfinder.php [hunk 2]
```diff
@@ -68,13 +68,25 @@ function add_addressfinder_widget( $_checkout ) {
...
+	add_filter( 'woocommerce_settings_addressfinder-settings', 'add_settings', 10, 1 );
```
Comment (vprigent) [inline woocommerce-addressfinder.php:78]: Adds the settings which is an array of array containing fields. Basically the same as before except we've extracted them out now.

File: woocommerce-addressfinder.php [hunk 3]
```diff
@@ -68,13 +68,25 @@ function add_addressfinder_widget( $_checkout ) {
...
 	}
 
-	add_filter( 'woocommerce_get_settings_checkout', 'addressfinder_settings', 10, 1 );
+	// Add the tab to the tabs array
+	function filter_addressfinder_settings_tabs_array( $settings_tabs ) {
+    $settings_tabs['addressfinder-settings'] = __( 'Addressfinder Settings', 'woocommerce' );
+
+    return $settings_tabs;
+	}
+	add_filter( 'woocommerce_settings_tabs_array', 'filter_addressfinder_settings_tabs_array', 99 );
```
Comment (bradleyannear) [inline woocommerce-addressfinder.php:77]: Whats `99`? Priority?

**Review by bradleyannear — APPROVED** [review]
LGTM

File: woocommerce-addressfinder.php (hunk 3 above)
Comment (vprigent) [reply woocommerce-addressfinder.php:77]: Order if I got that right. So that'll be the last tab item

**Pull request merged**